nautobot/utilities/testing/views.py

```diff
@@ -545,6 +546,35 @@ def test_delete_object_with_permission(self):
                 self.assertEqual(len(objectchanges), 1)
                 self.assertEqual(objectchanges[0].action, ObjectChangeActionChoices.ACTION_DELETE)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_delete_object_with_permission_and_xwwwformurlencoded(self):
```

Comment:
Wonder if we should add this to `CircuitTypeTestCase` or any test cases that use `NautobotUIViewSet` Views specifically? Lets wait to see if the test cases for others that use `generic.views` pass.

Reply:
The tests passed so I think we can keep it here